xbmc/cdrip/EncoderFFmpeg.cpp

```diff
@@ -60,14 +60,17 @@ bool CEncoderFFmpeg::Init()
     if (!m_formatCtx)
       throw FFMpegException("Could not allocate output format context");
 
-    m_bcBuffer = static_cast<uint8_t*>(av_malloc(BUFFER_SIZE + AV_INPUT_BUFFER_PADDING_SIZE));
-    if (!m_bcBuffer)
+    auto buffer = static_cast<uint8_t*>(av_malloc(BUFFER_SIZE + AV_INPUT_BUFFER_PADDING_SIZE));
```

Comment:
av_mallocz ?

Reply:
The documentation doesn't say that the buffer should be in any particular state so I think a regular `av_malloc` should be fine.